index.js

```diff
@@ -1,10 +1,13 @@
-const express = require('express'),
-  bodyParser = require('body-parser'),
-  session = require('express-session'),
-  cors = require('cors'),
-  errorhandler = require('errorhandler'),
-  log = require('fancy-log'),
-  swaggerUi = require('swagger-ui-express'),
+import express from 'express';
+import bodyParser from 'body-parser';
+import session from 'express-session';
+import cors from 'cors';
+import errorhandler from 'errorhandler';
+import log from 'fancy-log';
+
+import router from './server/routes';
+
+const swaggerUi = require('swagger-ui-express'),
   swaggerDocument = require('./server/docs/swagger.json');
```

Comment:
You can also use the import statement for swagger-ui-express package for consistency.

Reply:
Thanks for calling my attention on this